### PR TITLE
Feature/133 input values endpoint

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* [EQ-133](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=865&view=detail&selectedIssue=EQ-133)

## Main Changes:
* Added a `values` endpoint to the internal API, used for populating inputs in the interface. This endpoint will return column values that match a case-insensitive `LIKE` clause up to a specified limit, and optionally filter the results by other `WHERE` clauses.
* Updated the select inputs in the interface to send a request to the endpoint if field values are not already present in the domain values or list values.

## Steps To Test:
1. Start the client application, then navigate to the home page: https://localhost:3000/attains.
2. In the **Data Profile** drop-down, select *Assessment Units*.
3. The **Assessment Unit ID** field and the **Location Text** field both retrieve their values from the database. Confirm that they are populated with 20 values by default, and that the list of values updates as characters are entered into the input's search box.
4. Confirm that the **Location Text** field values update properly when different sources are selected from the source drop-down.

### CODE REVIEW

**Comment by maxdiebold-erg** [comment]
The values endpoint is currently crafted from two slugs (one for the profile, another for the column), in an attempt to mirror the structure of the endpoints on the `data` side. We could, however, just move it to a single `/data/values` endpoint, and put both the profile and column in the GET parameters.

Also, I didn't create a POST endpoint for the time being because this will only be used internally, GET requests are marginally faster, and there aren't many parameters.

**Review by cschwinderg — CHANGES_REQUESTED** [review]
Works well! 

I have one suggestion below.

File: app/client/src/config/index.tsx [hunk 1]
```diff
@@ -15,7 +15,10 @@ const { NODE_ENV, REACT_APP_CLOUD_SPACE } = process.env;
 // When deployed to Cloud.gov, the React app is built and served as static files
 // from the Express app, so it's one app running from a single port so no proxy
 // is needed for production.
-export const serverUrl = window.location.origin;
+export const serverUrl =
+  window.location.origin === 'localhost'
+    ? `${window.location.protocol}//${window.location.hostname}:9090`
+    : window.location.origin;
```
[the remark this answers is not in the document]
Comment (maxdiebold-erg) [reply app/client/src/config/index.tsx:21]: Oh, right!

**Review by cschwinderg — APPROVED** [review]
Thanks for making that change and fixing the merge conflicts.

**Pull request merged**